main.go

```diff
@@ -304,7 +308,7 @@ func setupKcpWatcherReconciler(mgr ctrl.Manager, options controller.Options, fla
 	// In total, we probably only have 20 watcher CRs, one worker can sufficiently handle it,
 	// and we don't have to deal with concurrent write to virtual service.
 	// although eventually the write operation will succeed.
-	options.MaxConcurrentReconciles = 1
```

Comment:
The comment can be deleted. After this issue https://github.com/kyma-project/lifecycle-manager/issues/302 is resolved, we don't need to limit workers for watcher, and also with your change, this comment is irrelevant anyway.